LibraryManagementSystem/Forms/IssueTab.cs

```diff
@@ -52,7 +53,65 @@ private void btnIssue_Click(object sender, EventArgs e)
             con.Close();
 
             txtBookID.Clear();
-            txtLibraryID.Clear();
+            txtLibraryID.Clear();*/
+
+            string libraryID = txtLibraryID.Text;
+            string unavailText = "Borrowed by student library ID: " + libraryID;
+
+
+            if (String.IsNullOrEmpty(txtBookID.Text) || String.IsNullOrEmpty(txtLibraryID.Text))
+            {
+                MessageBox.Show("Please complete all fields.", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
+            }
+            else {
+                bookID = Convert.ToInt32(txtBookID.Text);
+                cmd = new SqlCommand("SELECT * FROM booksData WHERE BookID = '" + bookID + "'", con);
+                da = new SqlDataAdapter(cmd);
+                dt = new DataTable();
+                da.Fill(dt);
+
+                if (dt.Rows.Count > 0)
+                {
+                    /*
+                    cmd = new SqlCommand("UPDATE booksData SET Status = '" + unavailText + "' WHERE BookID = '" + bookID + "'", con);
+                    cmd.ExecuteNonQuery();
+
+                    MessageBox.Show("Successfully issued book ID " + bookID + " to student " + libraryID, "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    con.Close();
+
+                    txtBookID.Clear();
+                    txtLibraryID.Clear();
+                    */
+
+                    cmd = new SqlCommand("SELECT * FROM booksData WHERE BookID = '" + bookID + "' AND Status = 'avail'", con);
+                    da = new SqlDataAdapter(cmd);
+                    dt = new DataTable();
+                    da.Fill(dt);
+
+                    if (dt.Rows.Count > 0)
+                    {
+                        cmd = new SqlCommand("UPDATE booksData SET Status = '" + unavailText + "' WHERE BookID = '" + bookID + "'", con);
+                        cmd.ExecuteNonQuery();
+
+                        MessageBox.Show("Successfully issued book ID " + bookID + " to member " + libraryID, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information);
+                        con.Close();
+
+                        txtBookID.Clear();
```

Comment:
Replace code for clearing textboxes in lines 99-100, 104-105, and 111-112.
From:
<kbd>txtBookID.Clear();
txtLibraryID.Clear();</kbd>
To only:
<kbd>btnClear_Click(sender, e);</kbd>